src/libmscore/measure.cpp

```diff
@@ -3926,7 +3926,10 @@ void Measure::setEndBarLineType(BarLineType val, int track, bool visible, QColor
         // no suitable bar line: create a new one
         bl = new BarLine(score());
         bl->setParent(seg);
-        bl->setTrack(track);
+        bl->setTrack(track); 
+        Part* part = score()->staff(track / VOICES)->part();
+        if (part != NULL && part->nstaves() > 1)
```

Comment:
Code style issue (for master) braces are mandatory

Reply:
Also use either `part !=nullptr` (NULL is C, not C++) or (better IMHO) just `part`.

Reply:
Thanks, anyway, will have to close this one and create a new one with the right branch.

Reply:
But... is the fix #xxxxxx supposed to refer to an issue on musescore.org or on github?

Reply:
Ypu can git rebase -i them.away

Reply:
on musescore.org